assets/js/modules/analytics-4/components/widgets/TopCategoriesWidget.js

```diff
@@ -107,7 +107,12 @@ function TopCategoriesWidget( { Widget } ) {
 			field: 'dimensionValues',
 			Component: ( { fieldValue } ) => {
 				const [ categories ] = fieldValue;
-				let categoriesList = JSON.parse( categories.value );
+
+				let categoriesList =
+					typeof categories?.value === 'string'
+						? categories?.value.split( '; ' )
+						: [];
```

Comment:
This can be a `const` as the result will always be an array (if `typeof categories?.value === 'string'` we know it's a string that we can call `split()` on, otherwise it's an empty array). Lines 116-118 below can then be removed.

```suggestion
				const categoriesList =
					typeof categories?.value === 'string'
						? categories.value.split( '; ' )
						: [];
```

Reply:
Easily done, we all miss these things from time to time :)